configuration-client/src/components/Settings/Settings.tsx

```diff
@@ -164,38 +164,53 @@ export default function Settings() {
 
   const moveTranscodingEnginesPriority = (purpose:number, oldIndex:number, newIndex:number) => {
     if (form.getInputProps('engines_priority').value instanceof Array<string>) {
-      var items = form.getInputProps('engines_priority').value as Array<string>;
+      let items = form.getInputProps('engines_priority').value as Array<string>;
       let index = items.indexOf(getTranscodingEnginesPriority(purpose)[oldIndex]);
       let moveTo = index - oldIndex + newIndex;
       form.setFieldValue('engines_priority', arrayMove(items, index, moveTo));
     }
   }
 
-  const getTranscodingEngineStatus = (engine: any) => {
-    if (engine.enabled && engine.isAvailable) {
-      return (
-        <Tooltip label={allowHtml(i18n.get['TranscodingEngineXEnabled']?.replace('%s', engine.name))} {...defaultTooltipSettings}>
-          <PlayerPlay strokeWidth={2} color={'green'} size={14}/>
-        </Tooltip>
-      )
-    }
-
-    if (!engine.enabled) {
-      return (
-        <Tooltip label={allowHtml(i18n.get['TranscodingEngineXDisabled']?.replace('%s', engine.name))} {...defaultTooltipSettings}>
-          <Ban color={'red'} size={14}/>
-        </Tooltip>
-      )
+  const setTranscodingEngineStatus = (id: string, enabled: boolean) => {
+    let items = (form.getInputProps('engines').value instanceof Array<string>) ?
+      form.getInputProps('engines').value as Array<string> :
+      [form.getInputProps('engines').value];
+    let included = items.includes(id);
+    if (enabled && !included) {
+	  let updated = items.concat(id);
+      form.setFieldValue('engines', updated);
+    } else if (!enabled && included) {
+      let updated = items.filter(function(value){ return value !== id;});
+      form.setFieldValue('engines', updated);
     }
+  }
 
+  const getTranscodingEngineStatus = (engine: {id:string,name:string,isAvailable:boolean,purpose:number,statusText:string[]}) => {
+    let items = (form.getInputProps('engines').value instanceof Array<string>) ?
+      form.getInputProps('engines').value as Array<string> :
+      [form.getInputProps('engines').value];
     if (!engine.isAvailable) {
       return (
         <Tooltip label={allowHtml(i18n.get['ThereIsProblemTranscodingEngineX']?.replace('%s', engine.name))} {...defaultTooltipSettings}>
           <ExclamationMark color={'orange'} strokeWidth={3} size={14}/>
         </Tooltip>
       )
+    } else if (items.includes(engine.id)) {
+      return (
+        <Tooltip label={allowHtml(i18n.get['TranscodingEngineXEnabled']?.replace('%s', engine.name))} {...defaultTooltipSettings}>
+          <ActionIcon size={10} style={{ cursor: 'copy' }} onClick={(e: any) => {setTranscodingEngineStatus(engine.id, false); e.stopPropagation();}}>
```

Comment:
I would recommend the ActionIcon size at least 20 because the size 10 is really small and it is not clear that clicking on it would change the state.

Reply:
The same is for the icons for moving the engine up or down.

Reply:
Done